cpp/include/Ice/LocalExceptions.h

```diff
@@ -788,7 +788,7 @@ namespace Ice
         [[nodiscard]] const char* ice_id() const noexcept final;
     };
 
-    /// This exception indicates a failure in a security subsystem, such as the IceSSL plug-in.
+    /// The exception that is thrown when a failure occurs in the security subsystem; this includes IceSSL errors.
```

Comment:
`;` is not correct here as its not separating two independent clauses. I would write either:

> The exception that is thrown when a failure occurs in the security subsystem, including IceSSL errors.

> The exception that is thrown when a failure occurs in the security subsystem. This includes IceSSL errors.

Reply:
Picked second option.